lib/src/models/chat_feedback_info.dart

```diff
@@ -0,0 +1,15 @@
+class ChatFeedbackInfo {
```

Comment:
All of the other chat-related models are currently in `lib/src/models/chat.dart`, so how about we move this there to keep them together?

Reply:
Okay, I'll move it